Libs/MRML/Core/vtkITKTransformConverter.h

```diff
@@ -341,8 +341,17 @@ bool vtkITKTransformConverter::SetVTKBSplineParametersFromITKGeneric(
     return false;
     }
 
-  typename BSplineTransformType::Pointer bsplineItk = dynamic_cast< BSplineTransformType* >( warpTransformItk.GetPointer() );
-  if (!bsplineItk.GetPointer())
+  typename BSplineTransformType::Pointer bsplineItk;
+  std::string warpTransformItkName = warpTransformItk->GetNameOfClass();
+  if (warpTransformItkName == "InverseBSplineTransform" ||
+      warpTransformItkName == "BSplineTransform" ||
+      warpTransformItkName == "BSplineDeformableTransform" ||
+      warpTransformItkName == "InverseBSplineDeformableTransform"
```

Comment:
The transforms that 3 template parameters do we have any guarantee that the ParamterType? Dimension or Spline order match?